docs/pom.xml

```diff
@@ -38,4 +38,61 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>release</id>
```

Comment:
Is there a reason you needed to add the release profile here? I would thought it got inherited from ddf-parent or are you overriding something here?